__tests__/database.test.js

```diff
@@ -0,0 +1,42 @@
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+const request = require("supertest");
+const app = require("../app");
+const {
+  categoryData,
+  commentData,
+  reviewData,
+  userData,
+} = require("../db/data/development-data/index");
+
+beforeEach(() => seed({ categoryData, commentData, reviewData, userData }));
+afterAll(() => db.end());
+
+describe("/api/categories", () => {
+  describe("GET", () => {
+    it("GET - status 200: responds with array of category objects", () => {
+      return request(app)
+        .get("/api/categories")
+        .expect(200)
+        .then((category) => {
+          expect(category.body.length).toBe(7);
```

Comment:
you aren't checking what you are sending back from your controller. 

You are sending back your categories on a key of  "categories". In your tests this would be res.body.categories for example

category.body is just checking the body property of the response that has come back - NOT the specific categories property that you have sent back